tools/nuget/ebpf-for-windows.nuspec.in

```diff
@@ -33,6 +33,7 @@
 		<file src="usersim.dll" target="build\native\bin"/>
 		<file src="usersim.pdb" target="build\native\bin"/>
 		<file src="..\..\include\**\*.*" target="build\native\include" />
+		<file src="..\..\docs\TroubleShootingGuide.html" target="."/>
```

Comment:
Since we are generating the html file only in CICD, and nuget package is generated as part of the local build also, will msbuild on dev machine fail post this change?

Reply:
+1

Reply:
I did the msbuild, the build was successful.

msbuild /m /p:Configuration=Debug /p:Platform=x64 ebpf-for-windows.sln

 Is there any specific command option to try?

Reply:
I think the nuget package is created only for the release build. Can you try the release build locally?

Reply:
> think the nuget package is created only for the release build

Yes, msbuild fails inRelease, with file not found error.

Reply:
> nuget package is created only for the release build

You are right. msbuild in Release failed, with file not found error.